task3/src/services/userService.js

```diff
@@ -0,0 +1,65 @@
+import { Op } from 'sequelize';
+
+export default class UserService {
+    constructor(model) {
+        this.model = model;
+    }
+
+    async create(userItem) {
+        try {
+            const data = await this.model.create(userItem);
```

Comment:
Why do you need to wait for promise resolution here & then once again to wait it in controller? You can simple return promise here 
```
create(userItem) {
    return this.model.create(userItem)
}

Reply:
I see, but what about update method ?
` async update(userItem) {
        try {
            const item = await this.model.findByPk(userItem.id);
            const updatedItem = await item.update(userItem);
            return updatedItem;
        } catch (err) {
            throw err;
        }
    }`
Am I making it less readable by changing async/await into Promises ?

Reply:
Update method can be left as is, I just meant, that my notation will bring you to exactly the same result with much less code lines. You'll still use you service with exactly the same way

Reply:
👍